src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.main__container {
+  width: 944px;
+  display: flex;
+  padding: 50px 40px;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
```

Comment:
Move this code to separate `main.scss` in `blocks` folder class. Because it's not a card block styles, it's `main` block styles.